ckanext/rating/templates/package/search.html

```diff
@@ -0,0 +1,21 @@
+{% ckan_extends %}
+
+{% block form %}
+  {% set facets = {
+    'fields': c.fields_grouped,
+    'search': c.search_facets,
+    'titles': c.facet_titles,
+    'translated_fields': c.translated_fields,
+    'remove_field': c.remove_field }
+  %}
+  {% set sorting = [
+    (_('Relevance'), 'score desc, metadata_modified desc'),
+    (_('Name Ascending'), 'title_string asc'),
+    (_('Name Descending'), 'title_string desc'),
+    (_('Rating Ascending'), 'rating asc'),
+    (_('Rating Descending'), 'rating desc'),
```

Comment:
Rating-related facets should be also hidden depending on the enabled_dataset_types-configuration